ffac-autoupdater-wifi-fallback/luasrc/usr/lib/lua/autoupdater-wifi-fallback/util.lua

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/lua
+local uci = require('simple-uci').cursor()
+local iwinfo = require 'iwinfo'
+local log = require 'posix.syslog'
+local M = {}
+
+function M.log(dest, msg)
+	local prefix = 'autoupdater-wifi-fallback: '
+	msg = prefix .. msg
+	if dest == 'out' then
+		io.stdout:write(msg .. '\n')
+		log.syslog(log.LOG_INFO, msg)
+	elseif dest == 'err' then
+		io.stderr:write(msg .. '\n')
+		log.syslog(log.LOG_CRIT, msg)
+	end
+end
+
+function M.get_available_wifi_networks()
+	local radios = {}
+
+	uci:foreach('wireless', 'wifi-device',
+		function(s)
+			radios[s['.name']] = {}
+		end
+	)
+
+	for radio, _ in pairs(radios) do
+		local wifitype = iwinfo.type(radio)
+		local iw = iwinfo[wifitype]
+		if not iw then
+			return null
+		end
+		local tmplist = iw.scanlist(radio)
+		for _, net in ipairs(tmplist) do
+			if net.ssid and net.bssid and net.ssid:match('.*[Ff][Rr][Ee][Ii][Ff][Uu][Nn][Kk].*') then
```

Comment:
maybe something like the following could work

```suggestion
			if net.ssid and net.bssid
			  and net.ssid:match('.*[Ff][Rr][Ee][Ii][Ff][Uu][Nn][Kk].*') then
```

Reply:
this line is not a problem in luacheck..? It is the other regex in line 50

Reply:
oh, in that case try something similar there